.changeset/rule-promotion.md

```diff
@@ -0,0 +1,85 @@
+- [style/noHeadElement](https://next.biomejs.dev/linter/rules/no-head-element)
+- [style/noNestedTernary](https://next.biomejs.dev/linter/rules/no-nested-ternary)
+- [style/noProcessEnv](https://next.biomejs.dev/linter/rules/no-process-env)
+- [style/noRestrictedImports](https://next.biomejs.dev/linter/rules/no-restricted-imports)
+- [style/noRestrictedTypes](https://next.biomejs.dev/linter/rules/no-restricted-types)
+- [style/noSubstr](https://next.biomejs.dev/linter/rules/no-substr)
+- [style/useAtIndex](https://next.biomejs.dev/linter/rules/use-at-index)
+- [style/useCollapsedIf](https://next.biomejs.dev/linter/rules/use-collapsed-if)
+- [style/useComponentExportOnlyModules](https://next.biomejs.dev/linter/rules/use-component-export-only-modules)
+- [style/useConsistentCurlyBraces](https://next.biomejs.dev/linter/rules/use-consistent-curly-braces)
+- [style/useConsistentMemberAccessibility](https://next.biomejs.dev/linter/rules/use-consistent-member-accessibility)
+- [style/useTrimStartEnd](https://next.biomejs.dev/linter/rules/use-trim-start-end)
+- [suspicious/noDocumentCookie](https://next.biomejs.dev/linter/rules/no-document-cookie)
+- [suspicious/noDocumentImportInPage](https://next.biomejs.dev/linter/rules/no-document-import-in-page)
+- [suspicious/noDuplicateElseIf](https://next.biomejs.dev/linter/rules/no-duplicate-else-if)
+- [suspicious/noHeadImportInDocument](https://next.biomejs.dev/linter/rules/no-head-import-in-document)
+- [suspicious/noIrregularWhitespace](https://next.biomejs.dev/linter/rules/no-irregular-whitespace)
+- [suspicious/noOctalEscape](https://next.biomejs.dev/linter/rules/no-octal-escape)
+- [suspicious/noTemplateCurlyInString](https://next.biomejs.dev/linter/rules/no-template-curly-in-string)
+- [suspicious/useAdjacentOverloadSignatures](https://next.biomejs.dev/linter/rules/use-adjacent-overload-signatures)
+- [suspicious/useGoogleFontDisplay](https://next.biomejs.dev/linter/rules/use-google-font-display)
+- [suspicious/useGuardForIn](https://next.biomejs.dev/linter/rules/use-guard-for-in)
+- [suspicious/useStrictMode](https://next.biomejs.dev/linter/rules/use-strict-mode)
+
+Moreover, the following JavaScript rules are now recommended:
+
+- [complexity/noUselessUndefinedInitialization](https://next.biomejs.dev/linter/rules/no-useless-undefined-initialization)
+- [complexity/useArrayLiterals](https://biomejs.dev/linter/rules/use-array-literals)
+- [correctness/noConstantMathMinMaxClamp](https://next.biomejs.dev/linter/rules/no-constant-math-min-max-clamp)
+- [correctness/noUnusedImports](https://biomejs.dev/linter/rules/no-unused-imports)
+- [correctness/noUnusedPrivateClassMembers](https://biomejs.dev/linter/rules/no-unused-private-class-members) (recommended by ESLint)
+- [correctness/noUnusedVariables](https://biomejs.dev/linter/rules/no-unused-variables) (recommended by ESLint)
+- [complexity/useDateNow](https://biomejs.dev/linter/rules/use-date-now)
```

Comment:
Ca we recommend https://biomejs.dev/linter/rules/no-unused-function-parameters/ too ? It's the sister of `noUnusedVariables`

Reply:
It is already recommended. It seems that I forget to add it to the list.